### PR TITLE
create new-branch and asign the changes

### PR DESCRIPTION
**Pull request by ylruiz**
The new-branch was created.

### CODE REVIEW

**Comment by ylruiz** [comment]
The new-branch was created.

File: js/contact.js [hunk 1]
```diff
@@ -1,38 +1,55 @@
...
-  res.status(200)
-     .sendFile(path.join(dir + '/index.html'));         
+  queryName = req.query.name;
+  if (queryName == undefined) {
+    queryName = "";
+  } 
+  var html =`<form action="/contact-us" method="post"> 
+               Enter your name*: 
+               <br> 
+               <input type="text" name="name" placeholder="Full name" value=${queryName}>
```
Comment (krvajal) [inline js/contact.js]: This does not work if the name contains spaces, you should put into quotes
```suggestion
               <input type="text" name="name" placeholder="Full name" value="${queryName}"> 
```

File: js/contact.js [hunk 2]
```diff
@@ -1,38 +1,55 @@
...
+    queryName = "";
+  } 
+  var html =`<form action="/contact-us" method="post"> 
+               Enter your name*: 
+               <br> 
+               <input type="text" name="name" placeholder="Full name" value=${queryName}> 
+               <br> 
+               Enter your email*: 
+               <br> 
+               <input type="email" name="emailAddress" placeholder="example@gmail.com">
```
Comment (krvajal) [inline js/contact.js]: Just use `email` as the name prop

File: js/contact.js [hunk 3]
```diff
@@ -1,38 +1,55 @@
...
+  var html = "";
+  if (nameBody == "" || email == "" ){
+    html =`<form action="/contact-us" method="post"> 
+               Enter your name*: 
+               <br> 
+               <input type="text" name="name" placeholder="Full name" value=${queryName}> 
+               <br> 
+               Enter your email*: 
+               <br> 
+               <input type="email" name="emailAddress" placeholder="example@gmail.com">
```
Comment (krvajal) [inline js/contact.js]: Just use `email` as the name prop

File: js/contact.js [hunk 4]
```diff
@@ -1,38 +1,55 @@
...
-  
-
-// create my dir
-/* const dir = '/home/raimelmedina/Documents/yunet/webserver/';
-
-// configure body-parser
-app.use(express.static(path.join(dir)));
-app.use(bodyParser.urlencoded({extended: true}));
-app.use(bodyParser.json()); */
+let queryName = "";
```
Comment (krvajal) [inline js/contact.js]: Why is this variable global here. It is not been shared by any other function. Keep the scope as limited as possible

**Pull request merged**